src/test/kotlin/com/exactpro/th2/common/schema/message/impl/rabbitmq/group/TestRabbitMessageGroupBatchRouter.kt

```diff
@@ -263,6 +257,75 @@ class TestRabbitMessageGroupBatchRouter {
         }
     }
 
+    @Nested
+    inner class BatchPublishing {
+
+        private val router = createRouter(mapOf(
+            "test-pine" to QueueConfiguration(
+                routingKey = "",
+                queue = "subscribe",
+                exchange = "test-exchange",
+                attributes = listOf("subscribe")
+            ),
+            "test-pin1" to QueueConfiguration(
+                routingKey = "test",
+                queue = "",
+                exchange = "test-exchange",
+                attributes = listOf("publish"),
+                filters = listOf(
+                    MqRouterFilterConfiguration(
+                        metadata = listOf(
+                            FieldFilterConfiguration(
+                                fieldName = "message_type",
+                                expectedValue = "test-message",
+                                operation = FieldFilterOperation.EQUAL
+                            )
+                        )
+                    )
+                )
+            ),
+        ))
+
+        @Test
+        fun `publish batch if all messages passed`() {
+            router.send(
+                MessageGroupBatch.newBuilder()
+                    .addGroups(MessageGroup.newBuilder()
+                        .apply { this += message("test-message", Direction.FIRST, "test-alias1") }
```

Comment:
different aliases in one group look strange for th2. th2 message group must have the same session alias and direction. Please use message name/properties/protocol/fields for test